src/module/Acceleration.cpp

```diff
@@ -141,8 +141,8 @@ double QuasiLinearTheory::modify(double steplength, Candidate* candidate)
 }
 
 
-ParticleSplitting::ParticleSplitting(Surface *surface, int numSplits,
-		int	crossingThreshold, double minWeight, std::string counterid)
+ParticleSplitting::ParticleSplitting(Surface *surface, int	crossingThreshold, 
+	int numSplits, double minWeight, std::string counterid)
```

Comment:
Following the [contribution guideline](https://github.com/CRPropa/CRPropa3/blob/master/CONTRIBUTING.md), the number of splits should be called `numberSplits`.